src/java/org/apache/cassandra/db/compaction/CompactionManager.java

```diff
@@ -153,18 +153,18 @@ public CompactionMetrics getMetrics()
      */
     public RateLimiter getRateLimiter()
     {
-        setRate(DatabaseDescriptor.getCompactionThroughputMebibytesPerSec());
+        setRate(DatabaseDescriptor.getCompactionThroughputBytesPerSec());
         return compactionRateLimiter;
     }
 
     /**
      * Sets the rate for the rate limiter. When compaction_throughput is 0 or node is bootstrapping,
      * this sets the rate to Double.MAX_VALUE bytes per second.
-     * @param throughPutMiBPerSec throughput to set in MiB/s
+     * @param throughputBytesPerSec throughput to set in B/s
      */
-    public void setRate(final double throughPutMiBPerSec)
+    public void setRate(final double throughputBytesPerSec)
```

Comment:
nit: maybe just change this back to `setRate(double throughput)`, ditch the local `throughput`, and let the doc comment clarify the units?

Reply:
This was actually originally throughputMbPerSec.
https://github.com/apache/cassandra/commit/5bb4bab12f8edfef95ed13cbabf8c0f377986065#diff-f7dd0237c343649f70b7ec9fefd7f6941a40b5164fd6063dce00fc09d2c234a7L163

It is not in the `CompactionManagerMbean` but it is public so not sure whether I have to worry about it actually...

Reply:
My OCD made me keep and deprecate `setRate `public method and add  `setRateInBytes` I will push soon the latest version post-review